### PR TITLE
- Meta-tags not generating

### PR DESCRIPTION
**Pull request by moxiegirl**
- Removing bad redirect
- Adding 1.8.1 to release notes
- Also making sure to generate meta description / keywords

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

File: layouts/partials/header.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="{{ with .Description }}{{ . }}{{ else }}{{ with .Site.Params.description }}{{ . }}{{ end }}{{ end }}">
+    <meta name="keywords" content="{{ with .Keywords }}{{ . }}{{ else }}{{ with .Site.Params.keywords }}{{ . }}{{ end }}{{ end }}">
     <title>{{.Title }} </title>
```
Comment (thaJeztah) [inline layouts/partials/header.html]: Not related, but, also thinking of this now; I think we should add a fixed suffix after the title, e.g.

`{{.Title }} | Docker (Official?) Documentation`

Might help with SEO and overall

Comment (moxiegirl) [inline layouts/partials/header.html]: Right now the title is pulled into the tab.  So...I don't want to take up too much space with other things.

**Comment by thaJeztah** [comment]
LGTM, thanks @moxiegirl !

**Comment by moxiegirl** [comment]
author LGTM

**Comment by moxiegirl** [comment]
Thanks Seb!

**Pull request merged**